keyboards/projectkb/signature87/signature87.h

```diff
@@ -0,0 +1,50 @@
+#pragma once
+
+#include "quantum.h"
+
+#define KNO KC_NO
+
+#define LAYOUT_all( \
```

Comment:
Okay, so `LAYOUT_all` is:

![all](https://user-images.githubusercontent.com/18669334/69908956-bed02680-13a7-11ea-95c5-4897ea0cfaa4.png)

and `LAYOUT_wkl_ansi` is:

![wkl](https://user-images.githubusercontent.com/18669334/69908966-fdfe7780-13a7-11ea-9e5c-bd57cd6e23be.png)

At this point I'm thinking there should be an alias `#define LAYOUT_tkl_ansi_split_bs_rshift LAYOUT_all`, and `LAYOUT_wkl_ansi` renamed to `LAYOUT_tkl_ansi_wkl`, and modified for a 2u Backspace and 2.75u Right Shift. That will make it easier on our end to add additional (community) layouts without having to modify the existing layout macros or the keymaps that use them.

The changes to `LAYOUT_wkl_ansi` will require changes to the `winkeyless` keymap.

@fauxpark What do you think?

Reply:
I agree, I will update better layouts tomorrow for review. Thanks for pointing it out